tests/debugger/do_break.expected

```diff
@@ -8,17 +8,19 @@ Breakpoint 2 at tests/debugger/do_break.js:36 (in test() at line:20, col:1)
 Breakpoint 3 at tests/debugger/do_break.js:33 (in f() at line:31, col:3)
 Breakpoint 4 at tests/debugger/do_break.js:45 (in f() at line:43, col:1)
 (jerry-debugger) list
-1: tests/debugger/do_break.js:51
-2: tests/debugger/do_break.js:36 (in test() at line:20, col:1)
-3: tests/debugger/do_break.js:33 (in f() at line:31, col:3)
-4: tests/debugger/do_break.js:45 (in f() at line:43, col:1)
+=== Active breakpoints  ===
+ 1: tests/debugger/do_break.js:51
```

Comment:
I think it gives better visibility if there is an indent before the elements in the list.